src/js/components/put-away/PutAwayCheckPage.jsx

```diff
@@ -50,20 +50,18 @@ class PutAwayCheckPage extends Component {
 
   constructor(props) {
     super(props);
-    const {
-      putAway, pivotBy, expanded, location,
```

Comment:
Why this was removed? You could do: 
`const { initialValues: { putAway, pivotBy, expanded, location } } = this.props;`